test/resources/covid_symptoms.bsv

```diff
@@ -1,26 +1,134 @@
 #CUI|TUI|CODE|SAB|STR|PREF
```

Comment:
I think this file got moved a while ago into `ctakesclient/resources/` as part of exposing it in the API to cumulus.

Also, I think your unit tests might be failing because the bsv loader doesn't like empty lines?